systemsdk/easy-log-bundle/1.10/config/packages/dev/easy_log.yaml

```diff
@@ -0,0 +1,5 @@
+easy_log:
+    log_path: '%kernel.logs_dir%/%kernel.environment%-readable.log'
+    # Use below if you're running application in container and want to log in stdout
+    #log_path: 'php://stdout'
```

Comment:
I suggest you put this comment in your bundles documentation. The recipe is not a replacement for your documentation. 

Feel free to add a link to your docs.

Reply:
Ok, done.